src/main/java/spoon/reflect/visitor/ImportScanner.java

```diff
@@ -32,20 +32,16 @@
 	 * Computes import of a {@link spoon.reflect.declaration.CtElement}
 	 *
 	 * @return class imports computed by Spoon, it does not contain static imports
-	 * @deprecated This method signature will change to return void in order to improve encapsulation: Use getAllImports to get the imports after calling it.
 	 */
-	@Deprecated
-	Collection<CtTypeReference<?>> computeImports(CtElement element);
+	void computeImports(CtElement element);
 
 	/**
 	 * Computes import of a {@link spoon.reflect.declaration.CtType}
 	 * (represent a class).
 	 *
 	 * @return imports computed by Spoon, it can be CtTypeReference (for classes), but also CtFieldReference (static field) or CtExecutableReference (static methods)
-	 * @deprecated This method signature will change to return void in order to improve encapsulation: Use getAllImports to get the imports after calling it.
 	 */
-	@Deprecated
-	Collection<CtReference> computeAllImports(CtType<?> simpleType);
+	void computeAllImports(CtType<?> simpleType);
```

Comment:
for simplification, can we keep only `computeImports(CtElement element)` and remove this one?